doc/proposals/openshift-4.3/operator-testing-tool.md

```diff
@@ -0,0 +1,240 @@
+---
+title: operator-testing-tool
+authors:
+  - "@jmccormick2001"
+  - "@joelanford"
+reviewers:
+  - TBD
+  - "@joelanford"
+approvers:
+  - TBD
+creation-date: 2019-09-13
+last-updated: 2019-09-13
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - "/enhancements/this-other-neat-thing.md"  
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# operator-testing-tool
+
+
```

Comment:
```suggestion
[Background](#Background)
[Goals](#Goals)
[Design overview](#Design_overview)
[User facing usage](#User_facing_usage)
[Observations and open questions](#Observations_and_open_questions)
```

Reply:
Note after add it just run `$doctoc relativepath file` in order to get it update. See https://www.npmjs.com/package/doctoc